aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,41 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## April 11, 2022 - June 3, 2022
+
+### Communications
+
+- We're looking to improve the usability of the resources we're delivering, making the site easier to search, providing more and better examples, etc. We've got a summer intern coming on board soon to help with this.
+
+### Stay Current with the Latest Changes
+
+- **Page Header**
+  - Both [Figma](https://www.figma.com/file/BHp0KrOcPvbnzNyshfHkd8/HPE-Page-Header?node-id=734%3A40546) and [grommet](https://v2.grommet.io/pageheader) components are now available as well as [design guidance](https://design-system.hpe.design/components/pageheader)
+
+### Roadmap Updates
+
+- **Figma Component Refactoring**
+  - We are currently working on input components. Thanks for all of the help we've received from folks across HPE in this effort.
+- **Multi-Select**
+  - Wrapping up basic anatomy and focusing on how to represent the current selection.
+- **Adoption Scorecard**
+  - Working on initial draft version of a spreadsheet to evaluate how well products and services are aligned to the HPE Design System. Look for more to come in this area soon.
+
+### Latest Releases
+
+- **grommet v2.24.0** + **grommet-theme-hpe v3.1.0**
+  - Added a new **PageHeader** component.
+  - List now supports `disabled` items and will indicate how the use is interacting via `onActive`.
```

Comment:
I'm not following what "will indicate how the use is interacting" is trying to say

Reply:
Typo, should have been: "will indicate how the user is interacting". Fixed.